piecrust-uplink/src/abi/state.rs

```diff
@@ -295,3 +297,26 @@ where
         unsafe { ext::emit(topic_ptr, topic_len, arg_len) }
     });
 }
+
+/// Feeds the host with data.
+///
+/// This is only allowed to be called in the context of a `feed_call`, and
+/// will error out otherwise. It is meant for contracts to be able to report
```

Comment:
Should we put some smart constraints on this on the Rusk side? I can imagine this can be used to DDoS nodes.

Reply:
Definitely. Especially when offering these as a service to the wider web, at least a rate limit will be required. I'm unsure as to whether the appropriate place for this kind of defensive measure should be at the infrastructure level, or at the level of rusk itself. We'll see.